plugins/go.mod

```diff
@@ -4,14 +4,59 @@ go 1.20
 
 require golang.org/x/exp v0.0.0-20231110203233-9a3e6036ecaa
 
+require github.com/blueprint-uservices/blueprint/runtime v0.0.0
```

Comment:
This will automatically lead to no outside repository being able to use blueprint plugins.

We can't use the replace directive in packages that are intended to be used as libraries.